recipes/scons/scons.inc

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "A Software Construction Tool"
+LICENSE = "MIT"
+RECIPE_TYPES = "native"
+
+SRC_URI = "http://ftp.osuosl.org/pub/blfs/conglomeration/scons/scons-${PV}.tar.gz"
+
+do_compile() {
+    python setup.py build
```

Comment:
Not according to either the scons webpage or the Arch Linux package:
http://www.scons.org/doc/production/HTML/scons-user/ch01s03.html
https://www.archlinux.org/packages/extra/any/scons/